tests/test_sftp.py

```diff
@@ -28,14 +28,15 @@
 import sys
 import warnings
 from binascii import hexlify
+from io import StringIO
 from tempfile import mkstemp
 
 import pytest
 
-from paramiko.py3compat import PY2, b, u, StringIO
 from paramiko.common import o777, o600, o666, o644
-from tests import requireNonAsciiLocale
 from paramiko.sftp_attr import SFTPAttributes
+from paramiko.util import b, u
+from tests import requireNonAsciiLocale
```

Comment:
Nit - Why the initial lowercase 'r' on this otherwise camelcase?

Reply:
Whatever random contributor added that did it that way, apparently. This isn't worth changing now, I'd rather look harder at the underlying helper when we do a second cleanup pass, and either nix it entirely or rename it.